.github/workflows/build.yml

```diff
@@ -2,11 +2,33 @@ name: Android CI
 on:
   push:
 jobs:
-  build:
+  build-and-test:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+      - name: Checkout Yfirlestur Repo
+        uses: actions/checkout@v3
+        with:
+          repository: mideind/Yfirlestur
+          path: yfirlestur
+          ref: dockerfile_development
```

Comment:
I would base this on a commit SHA1 inside the master branch, not on the branch `dockerfile_development`. A branch can go away anytime, especially if it's a merge request branch. Whereas a SHA1 in the master branch is unlikely to go away (though not impossible).

Reply:
addressed in the most recent PR's where this has been merged to master on Yfirlestur.